docx-asciidoc-conversion/README.md

```diff
@@ -16,39 +17,42 @@ The procedure was tested using the INSPIRE Metadata Technical Guidance 2.0. The
 
 ### Conversion Procedure
 
-1. Correct the original docx document, save as `md_2_fixed.docx`:
-   - apply *XML Example* style to all XML examples (about 7 cases were found in `md_2_0_1.docx` where this fix was required)
-   - remove paragraph breaks from inside footnotes (two cases were found in `md_2_0_1.docx` where this fix was required)
+1. Create a new folder with the name of the document to convert:
+   ```shell
+    mkdir my_example_data_specifications_EX
+    ```
 
-2. Convert the corrected docx document to *AsciiDoc* using Pandoc, save as `md_2_pandoc.adoc`, extract images to `media` folder:
+2. Copy the Microsoft Word document into the new folder. Convert the Microsoft Word document into the last .docx format:
     ```shell
-    pandoc md_2_fixed.docx -f docx -t asciidoc -o md_2_pandoc.adoc --wrap=none --extract-media=. --markdown-headings=atx
+    From Microsoft Word document: File -> Informations -> Convert
```

Comment:
not sure this is a menu in Word

Reply:
The mentioned menù is present only if a .doc file is opened.

Reply:
Not sure this is the case, I tried with a .doc file and I don't see the menu. Also, 'Informations' look strange, since it should be 'Information' (it is an uncountable noun).

Reply:
Thanks for the check, there was an error since the name of the menù in the English version of MS Word is "Info". The error has been corrected (Commit: https://github.com/INSPIRE-MIF/technical-guidelines/commit/5c2977e6322c22a9913d1fe4ba22cc929cb06aa6).